images/dvcr-artifact/FUZZING.md

```diff
@@ -0,0 +1,42 @@
+# Uploader Package Fuzzing Tests
```

Comment:
Please, add a description of tests. There is no info about what exactly we test. I see a dvcr mock with endpoint "/v2/uploader/blobs/uploads/test_data", but it is unclear what is happening here.